README.md

```diff
@@ -66,7 +77,7 @@ Select 1:
 ```
 [] Selects all li's which are directly inside a ul of class dropdown (children)
 [] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
-[] Selects all ul's of class dropdown, as well as the children elements that are li's
+[x] Selects all ul's of class dropdown, as well as the children elements that are li's
 [] Selects all ul's of class dropdown, only if their children are exclusively li's
 ```
```

Comment:
Correct answer is the first option. Whenever you see `>`, it means "direct descendant".

The selector for the answer you chose would be...

```css
ul.dropdown, ul > li {
  /* properties go here */
}
```